sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -120,11 +121,13 @@ abstract class JdbcDialect extends Serializable {
    * The SQL query that should be used to truncate a table. Dialects can override this method to
    * return a query that is suitable for a particular database. For PostgreSQL, for instance,
    * a different query is used to prevent "TRUNCATE" affecting other tables.
-   * @param table The name of the table.
+   * @param table The table to truncate
+   * @param cascade Whether or not to cascade the truncation
    * @return The SQL query to use for truncating a table
    */
   @Since("2.3.0")
-  def getTruncateQuery(table: String): String = {
+  def getTruncateQuery(table: String,
+                       cascade: Option[Boolean] = isCascadingTruncateTable): String = {
```

Comment:
@danielvdende .
Could you take a look the guide? This guide is helpful in Apache Spark community.
- https://github.com/databricks/scala-style-guide

Reply:
For this one, we need to keep the original one for backward compatibility like the following?
```
  @Since("2.3.0")
  def getTruncateQuery(table: String): String = {
      getTruncateQuery(table, isCascadingTruncateTable)
  }

  @Since("2.4.0")
  def getTruncateQuery(table: String, cascade: Option[Boolean] = isCascadingTruncateTable)
```